### PR TITLE
Update javascript section to be accurate

### PR DESCRIPTION
**Pull request by kodypeterson**

### CODE REVIEW

File: source/includes/storing_data.md [hunk 1]
```diff
@@ -49,12 +49,12 @@ namespace Oxide.Plugins
...
 ``` javascript
 Init: function() {
-    this:LoadDataFiles();
+    this.LoadDataFiles();
 }
 
 var dataTable
 LoadDataFiles: function() {
-    dataTable = datafile.GetDataTable("pluginName") || {};
+    dataTable = data.GetData("pluginName");
```
Comment (kodypeterson) [inline source/includes/storing_data.md]: It seems oxide returns a reference to an internal object. In order to ensure that the data is saved, we need to always modify that reference.

Comment (lukespragg) [inline source/includes/storing_data.md]: I believe that was there to create an empty table if one didn't already exist. I've seen to done in the other plugin languages as well.

Comment (kodypeterson) [inline source/includes/storing_data.md]: What I am finding is that if one does not exist an empty object is being returned, as I would expect. So, you never get to the "or" part of this condition.

**Comment by lukespragg** [comment]
Thanks for the PR. :)

**Pull request merged**